hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/client/QuorumJournalManager.java

```diff
@@ -479,10 +479,9 @@ public void recoverUnfinalizedSegments() throws IOException {
     LOG.info("Successfully started new epoch " + loggers.getEpoch());
 
     if (LOG.isDebugEnabled()) {
```

Comment:
and here

Reply:
There is string concatenation in `QuorumCall.mapToString(resps)`, so we need to keep this judgment.

Reply:
Yes, reasonable!